packages/cra-template-seed/template/src/components/Form/Label/Label.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { spacing } from '@ui5/webcomponents-react-base';
+import { Label as UI5Label } from '@ui5/webcomponents-react';
+
+const Label = ({ style, ...props }) => {
+  const innerStyle = {
+    ...style,
+    ...spacing.sapUiTinyMarginBottom,
+    lineHeight: '20px',
+  };
+
+  return (
+    <UI5Label data-testid="label-wrapper" wrap style={innerStyle} {...props}>
```

Comment:
do you really need the testId?

Reply:
this one we need for test @ `Label.test.js` file.